dist/templates/k8s.ovn.org_egressfirewalls.yaml.j2

```diff
@@ -0,0 +1,122 @@
+
```

Comment:
Can you make sure that when you apply the CRD on a cluster, you can do `oc explain egressfirewalls.k8s.ovn.org` without any errors and the Spec fields show up correctly?

Also per openapiv3, do you need `preserveUnknownFields` tag in the CRD?

Reply:
I tested this for egress IP and it took me a while to understand, the problem is `controller-gen`. The output it produces generates the CRD template with `apiextensions.k8s.io/v1beta1`, it should be `apiextensions.k8s.io/v1`, moreover it only sets the `.spec.validation` part of the CRD. To get the explain output we'll need that part to be set on `.spec.versions.schema`. Take a look at this for a validated version that works with explain: https://github.com/ovn-org/ovn-kubernetes/pull/1484/commits/a6fb537c1f1c476d0f92ab808a11427b081c75c2

Reply:
I changed the generator to generate the right api version and it correctly adds the description fields

Reply:
The generation of the crd has been fixed